NSS-Day-Cohort-66/rare-client-rare-poultry
# Buttons and Modals added for Edit and Delete functionality (WIP)

# Buttons, Modals, and updated styling
## Steps to Implement
- Created custom delete and edit .png
- Replaced buttons with icons
- Added a `input-text` tailwind custom class for styling text inputs

## Steps to Test
- Fetch and check out the `nc-tag-edit` branch
- Navigate to the tag editor
- Click both edit and delete buttons next to tags, ensuring that you can open and close the modals.
